tests/continental_compression_with_imposed_faults.prm

```diff
@@ -0,0 +1,37 @@
+# Continental Compression with Imposed Faults Cookbook
+# This cookbook builds on the continental extension cookbook
+# and a recent paper investigating rift inversion (Vasey et al., 2024,
+# https://doi.org/10.1130/G51489.1) to demonstrate how to design
+# models of continental compression that extend into the asthenosphere
+# and include faults as zones of weakness in the initial conditions.
+# The setup is motivated by observed deformation patterns in the Mongolian
+# Altai, which has experienced low magnitudes of compressional deformation
+# since the onset of the India-Asia collision and contains evidence for
+# reactivation of normal faults formed during prior phases of deformation
+# The faults locations and properties are defined using the Geodynamic
+# World Builder.
```

Comment:
Refer to my top comment, this is what I wanted to see at the top of the cookbooks/.prm file.
The documentation in this cookbooks/.prm and the tests/.prm file should just be switched. I would still add a little more detail based on my comment on the test .prm file though